pyzeebe/channel/oauth_channel.py

```diff
@@ -0,0 +1,146 @@
+from functools import partial
+from typing import Optional
+
+import grpc
+from grpc.aio._typing import ChannelArgumentType  # type: ignore[import-untyped]
+
+from pyzeebe.credentials.oauth import Oauth2ClientCredentialsMetadataPlugin
+
+
+def create_oauth2_client_credentials_channel(
+    target: str,
```

Comment:
```suggestion
    hostname: Optional[str] = None,
    port: Optional[int] = None,
```
As other constructors

Reply:
We can do it like this, but: 
1. I don't think there is much of a benefit, users could just use `target=f"{hostname}:{port}`
2. Camunda Java Client only has one variable.
https://docs.camunda.io/docs/apis-tools/java-client/#bootstrapping

Reply:
Yes, but now we use host and port as two args in other methods. I think, if you want to merge it to one `target`, it should be in another PR.

Reply:
Ok, i will soon provide another PR... 🤖